integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkEnvIT.java

```diff
@@ -106,4 +105,8 @@ private void initSparkEnv() {
             .enableHiveSupport()
             .getOrCreate();
   }
+
+  protected abstract String getCatalogName();
```

Comment:
could you move the class before `getSparkSession`? normally the methods are placed in the `public` `protected` `private`  sequence.

Reply:
personally, I prefer to put the abstract method to the top level of the class, so other developers could easily know why it's abstract.

Reply:
Fixed and thanks for pointing out the shortcomings